test/appsync.test.ts

```diff
@@ -73,32 +39,24 @@ $util.qr($v1.put('stateMachineArn', '${machine.stateMachineArn}'))
       () => {}
     );
 
-    const templates = getAppSyncTemplates(
+    appsyncTestCase(
       reflect(() => {
         machine({ input: { id: "1" } });
-      })
-    );
-
-    appsyncVelocityJsonTestCase(
-      templates[1],
-      { arguments: {}, source: {} },
-      {
-        result: {
-          version: "2018-05-29",
-          method: "POST",
-          resourcePath: "/",
-          params: {
-            headers: {
-              "content-type": "application/x-amz-json-1.0",
-              "x-amz-target": "AWSStepFunctions.StartExecution",
-            },
-            body: {
-              stateMachineArn: machine.stateMachineArn,
-              input: JSON.stringify({ id: "1" }),
+      }),
+      [
+        {
+          index: 1,
+          expected: {
+            match: {
+              params: {
+                body: {
+                  stateMachineArn: machine.stateMachineArn,
+                },
+              },
```

Comment:
Is this necessary? Seems verbose and redundant?

Reply:
It lets us test that the CDK tokens are in the right place, because we strip them out for the snapshot.

Reply:
I could remove the expected wrapper.